src/middlewares/error-sender.js

```diff
@@ -9,13 +9,16 @@ const getCommonErrorResponseObject = error => ({
 });
 
 const errorSender = (error, requset, response, next) => {
+  let errorResponseObject;
+
   if (error instanceof BaseHttpError) {
-    error.send(response);
+    errorResponseObject = error.getResponseObject();
```

Comment:
@v1adko I need your advice. What is right way to do it, using `error.send` or this way? Earlier I do error.send, but now i think, that "sending error" it's not the right behavior of error instance. And besides this way more seems like DRY. :)

Reply:
Yeah, I agree considering how much extra hacks are needed to for this double inheritance and all of that.
Keep the errors classes minimal, clean them up altogether

Reply:
Good. Thank you.